02-ca-certificates.md

```diff
@@ -1,10 +1,10 @@
-# Generate Your Client-Facing TLS Certificate
+# Generate Your Client-Facing and AKS Ingress Controller TLS Certificates
 
-Now that you have the [prerequisites](./01-prerequisites) met, follow the steps below to create the TLS certificate that Azure Application Gateway will serve for clients connecting to your web app. If you already have access to an appropriate certificate, or can procure one from your organization, consider doing so and skipping step 1. The following will describe using a self-signed cert for instructive purposes only.
+Now that you have the [prerequisites](./01-prerequisites) met, follow the steps below to create the TLS certificates that Azure Application Gateway will serve for clients connecting to your web app as well as the AKS Ingress Controller. If you already have access to an appropriate certificates, or can procure them from your organization, consider doing so and skipping the certificate generation steps. The following will describe using a self-signed certs for instructive purposes only.
 
 ## Steps
 
-1. Generate a self-signed TLS certificate
+1. Generate a client-facing self-signed TLS certificate
 
    > :book: Contoso Bicycle needs to procure a CA certificate for the web site. As this is going to be a user-facing site, they purchase an EV cert from their CA.  This will serve in front of the Azure Application Gateway.  They will also procure another one, a standard cert, to be used with the AKS Ingress Controller. This one is not EV, as it will not be user facing.
 
```

Comment:
Need to be reviewed 
Line 13,
s, **a second TLS certificate is going to be generated, but that one will be generated directly from your Azure Key Vault.**

Reply:
good catch